### DISCUSSION TITLE
Using a ‘target’ as a shiny ‘reactive expression’ ?

### DISCUSSION THREAD

**Discussion by JohnStaples** [ropensci/targets#1022] [Help]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Is it possible to use a ‘target’ to establish an R shiny ‘reactive expression’ in order to eliminate the need of running a server?
With anticipatory thanks …

**Comment by wlandau** [comment]
Not sure I fully understand what you have in mind, but I do not think integration this granular is likely to succeed. But you can do things that clearly decouple the role of `targets` and the role of Shiny. E.g. you can have a `targets` data analysis pipeline with a Shiny app wrapper around it, as in https://github.com/wlandau/targets-shiny, Or you could have a `targets` pipeline that generates data for a Shiny app or the Shiny app itself, similar to the concept at https://github.com/ropensci/targets/discussions/1001.

**Reply by JohnStaples** [reply]
Many thanks for your reply; I appreciate the enclosed links ….
My interest is building an agricultural research app with Quarto, R - Shiny and ….??
I tend to think out-of-the-box …!!
Thank you.
